GoogleDataTransport/GDTCORLibrary/GDTCOREvent.m

```diff
@@ -25,6 +25,37 @@
 
 @implementation GDTCOREvent
 
++ (NSNumber *)nextEventID {
+  static NSInteger sessionSequenceNumber = 1;
```

Comment:
What's the point of initializing to one?  Isn't it always rewritten in the dispatch_once?

Reply:
Changed.